docs/methoddocs/high_risk_employee.md

```diff
@@ -0,0 +1,15 @@
+# High Risk Employee
```

Comment:
We should also include an` autoclass` for `DetectionListUserClient`

Reply:
all of its methods are exposed on `DetectionListModule`, so no need

Reply:
`DetectionListUserClient` could be made internal.

Reply:
Let's make it internal!